api/environments/views.py

```diff
@@ -189,7 +184,7 @@ def permissions(self, *args, **kwargs):
         url_name="my-permissions",
     )
     def user_permissions(self, request, *args, **kwargs):
-        if request.user.is_anonymous:
+        if hasattr(request.user, "is_master_api_key_user"):
```

Comment:
This should probably be a `getattr` too? There are a few more cases where we're doing this exact check too. I don't think it makes sense to be `hasattr`, right? Because (based on what we talked about earlier regarding the property), it could in some future scenario return `False`, right?

Reply:
Yeah, done